website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4519,6 +4519,8 @@ registered, then Beam will automatically infer the coder for the state value. Ot
 specified when creating the ValueState. For example, the following ParDo creates a  single state variable that
 accumulates the number of elements seen.
 
+Note: ValueState is called ReadModifyWriteState in Python SDK.
```

Comment:
Code font `ValueState` and `ReadModifyWriteState`

Reply:
Done.